app/models/post.rb

```diff
@@ -1,2 +1,24 @@
+require 'csv'
+require 'prawn'
 class Post < ApplicationRecord
+  belongs_to :user
```

Comment:
Add an empty line after this line, this is due to readability reasons, it is easy to read when blocks of methods are separated by empty lines.